deps.edn

```diff
@@ -195,22 +198,21 @@
    {clj-http-fake/clj-http-fake  {:mvn/version "1.0.3"
                                   :exclusions  [slingshot/slingshot]}
     clj-kondo/clj-kondo          {:mvn/version "2022.08.03"}                ; this is not for RUNNING kondo, but so we can hack on custom hooks code from the REPL.
-    cloverage/cloverage          {:mvn/version "1.2.2"}
+    cloverage/cloverage          {:mvn/version "1.2.4"}
     com.gfredericks/test.chuck   {:mvn/version "0.2.13"}                    ; generating strings from regexes (useful with malli)
-    djblue/portal                {:mvn/version "0.30.0"}                    ; ui for inspecting values
```

Comment:
did you mean to remove the comments here?

Reply:
Oops, no, that was `antq`. Will restore them.